aces/Loris
# [Tools] Enable PHPCS for fix_timepoint_date_problems.php

Replaces #5984

This PR also makes changes to the User and Timepoint class, to fix return type inaccuracies.

## To test
- checkout the branch
- run PHPCS and check that there is no errors: 
  `npm run lint:php `
- run the tool and check that there is no errors: 
  `php tools/fix_timepoint_date_problems.php `
